scylla-server/Cargo.toml

```diff
@@ -9,10 +9,10 @@ diesel = { version = "2.2.4", features = ["postgres", "r2d2", "chrono"] }
 pq-sys = { version = "0.6", features = ["bundled"] }
 dotenvy = "0.15"
 serde = "1.0.203"
-protobuf-codegen = "3.5.1"
-protobuf = "3.5.1"
+protobuf-codegen = "3.7.1"
+protobuf = { version = "3.7.1", features = ["with-bytes"] }
```

Comment:
Just a general question... does protobuf release have to be synced across or can you just be on the whole number version and it will be compatible?

Reply:
Good question.  I think their site says within a single major version, so as long as all runtime libraries are 3.x we are good.

Reply:
https://protobuf.dev/support/cross-version-runtime-guarantee/